code/game/machinery/computer/guestpass.dm

```diff
@@ -6,34 +6,14 @@
 	desc = "Allows temporary access to station areas."
```

Comment:
Are the guest IDs used anywhere? If not, you can safely remove it entirely.

Reply:
They are located in big red